### PR TITLE
Add functional jobs to config map

### PR DESCRIPTION
**Pull request by cescgina**
Add tox-based functional jobs following the same requirements used until
now, that is, only py39+ and excluding storage jobs.

### CODE REVIEW

File: znoyder/config.yml [hunk 1]
```diff
@@ -53,64 +62,77 @@ include:
 exclude:
   'cinder':  # OSPCRE-22
     'osp-17.0':
-      'osp-tox-pep8': 'Storage DFG decided to opt-out for now'
-      'osp-tox-py39': 'Storage DFG decided to opt-out for now'
+      'openstack-tox-pep8': 'Storage DFG decided to opt-out for now'
+      'openstack-tox-py39': 'Storage DFG decided to opt-out for now'
+      'osp-tox-functional-py39': 'Storage DFG decided to opt-out for now'
```
Comment (sdatko) [inline znoyder/config.yml:67]: All the `openstack-tox-` should be `osp-tox-` here and below

**Review by sdatko — CHANGES_REQUESTED** [review]
Thanks for this. I left a comment, please update. Then, before merging, we will test the jobs like the last time :-)

File: znoyder/config.yml (hunk 1 above)
Comment (cescgina) [reply znoyder/config.yml:67]: Oh, it's true, I saw it in your PR, but I forgot to update, I'll do it now

**Review by IgnacioPro — APPROVED** [review]
LGTM!

**Review by bregman-arie — APPROVED** [review]
Looks good. Thanks for adding these jobs. Let's make sure we announce it somehow to the relevant teams.

**Review by sdatko — APPROVED** [review]
Some entries are actually providing no jobs right now for OSP 17.0, but we will deal about that later...

**Pull request merged**